xbmc/network/upnp/UPnPInternal.cpp

```diff
@@ -1015,8 +1015,8 @@ CFileItemPtr BuildObject(PLT_MediaObject* entry,
         UPNP::PopulateTagFromObject(*pItem->GetMusicInfoTag(), *entry, res, upnp_service);
 
     } else if(image) {
-        //CPictureInfoTag* tag = pItem->GetPictureInfoTag();
-
+      //! @todo fill pictureinfotag?
```

Comment:
Did you mean to include this feature in the PR?

Reply:
no, I just noticed that for all other content kodi populates the infotag from the received `PLT_MediaObject` while for pictures we don't (and we have that line of code commented). Not sure if there is any info from the response that might be usable/important for upnp pictures since they are a bit different. Maybe some exif tags might be pre-populated, hence decided to keep a todo there. Doesn't necessarily mean I'll do it later though.